src/consts.rs

```diff
@@ -44,5 +44,10 @@ pub const UHYVE_PORT_NETSTAT: u16 = 0x700;
 pub const UHYVE_PORT_CMDSIZE: u16 = 0x740;
 pub const UHYVE_PORT_CMDVAL: u16 = 0x780;
 
-pub const UHYVE_UART_PORT: u16 = 0x800;
+#[cfg(target_arch = "x86_64")]
+pub const UHYVE_UART_PORT: u16 = 0x3F8;
+
+#[cfg(target_arch = "aarch64")]
+pub const UHYVE_UART_PORT: u16 = 0x1000;
```

Comment:
Why `0x1000` instead of `0x800`?

Reply:
Because that's what the bootloader uses natively. But we need to talk about this PR anyway before merging.